log4j-api/src/main/java/org/apache/logging/log4j/message/FlowMessageFactory.java

```diff
@@ -30,6 +40,25 @@ public interface FlowMessageFactory {
      */
     EntryMessage newEntryMessage(Message message);
 
+    /**
+     * Creates a new exit message based on a return value and a forma string.
+     *
+     * @param format a format string
+     * @param result the return value
+     * @return the new exit message
+     * @since 2.20
+     */
+    ExitMessage newExitMessage(String format, Object result);
```

Comment:
Should these be default methods, or is it ok to break the API here?

Reply:
I am not sure what the default implementation should be.

Anyway a minor version bump allows to break a SPI.

Reply:
That sounds reasonable. Sometimes, a default implementation could be to either return `null` or just `throw new UnsupportedOperationException()` with appropriate checks on the consuming end as appropriate.